addons/dragging/functions/fnc_carryObject.sqf

```diff
@@ -26,6 +26,9 @@ private _direction = _target getVariable [QGVAR(carryDirection), 0];
 
 // handle objects vs persons
 if (_target isKindOf "CAManBase") then {
+    if (!alive _target) then {
+        _target = [_target] call FUNC(createClone);
+    };
```

Comment:
```suggestion
```
looks like this call is not necessary because there is another one in `fnc_startCarry` below. Not tested